diff --git a/bundle/ngx_lua-0.10.20/src/ngx_http_lua_ssl_certby.c b/bundle/ngx_lua-0.10.20/src/ngx_http_lua_ssl_certby.c
index 6ed2f3f..1ee2d28 100644
--- a/bundle/ngx_lua-0.10.20/src/ngx_http_lua_ssl_certby.c
+++ b/bundle/ngx_lua-0.10.20/src/ngx_http_lua_ssl_certby.c
@@ -1350,6 +1350,13 @@ int
 ngx_http_lua_ffi_ssl_verify_client(ngx_http_request_t *r, void *ca_certs,
     int depth, char **err)
 {
+#ifdef LIBRESSL_VERSION_NUMBER
+
+    *err = "LibreSSL not supported";
+    return NGX_ERROR;
+
+#else
+
     ngx_http_lua_ctx_t          *ctx;
     ngx_ssl_conn_t              *ssl_conn;
     ngx_http_ssl_srv_conf_t     *sscf;
@@ -1466,6 +1473,7 @@ failed:
     X509_STORE_free(ca_store);
 
     return NGX_ERROR;
+#endif
 }
 
 
